tests/functional/monitoring/libtest/test_workload_fixture.py

```
@@ -62,67 +62,140 @@ def test_workload_rbd(workload_storageutilization_50p_rbd, threading_lock):
         step=15,
     )
+    logger.info(f"OSD capacity query returned {len(result_total)} time series")
+
+    logger.test_step("Validate OSD capacity consistency")
     # Check test assumption that ceph_osd_stat_bytes hasn't changed for each
     # OSD, and that each OSD has the same size.
     osd_stat_bytes = []
-    for metric in result_total:
+    for i, metric in enumerate(result_total, 1):
         values = []
         for ts, value in metric["values"]:
```

Ruff (0.15.17)

[warning] Loop control variable `ts` not used within loop body

(B007)